src/components/Banner/GalleryList.jsx

```diff
@@ -36,43 +39,35 @@ const ImageSection = () => {
       });
   }, []);
 
-  const toggleFavourite = (photo) => {
+  const handleToggleFavourite = (photo) => {
     const currentFavourites = getFavFromLocalStorage();
-    const updatedFavourites = currentFavourites.some(
-      (fav) => fav.id === photo.id
-    )
-      ? currentFavourites.filter((fav) => fav.id !== photo.id)
-      : [...currentFavourites, photo];
-    saveFavInLocalStorage(updatedFavourites);
-    setPhotos((prevPhotos) =>
-      prevPhotos.map((p) =>
-        p.id === photo.id ? { ...p, favourite: !p.favourite } : p
-      )
+    const updatedFavourites = toggleFavourite(photo, currentFavourites);
+    const updatedPhotos = photos.map((p) =>
+      p.id === photo.id ? { ...p, favourite: !p.favourite } : p
     );
+    setPhotos(updatedPhotos);
   };
 
   const handleTabChange = (tab) => {
```

Comment:
Should we need this function and this props anymore?

Reply:
sePhotos is necessary

Reply:
handkeTabChange is removed